environments/production/terraform.yml

```diff
@@ -279,6 +279,24 @@ servers:
     group: "kafka"
     os: ubuntu_pro_bionic
 
+  - server_name: "kafka4-production"
+    server_instance_type: t3.medium
+    network_tier: "db-private"
+    az: "b"
+    volume_size: 300
+    volume_encrypted: no
+    group: "kafka"
+    os: ubuntu_pro_bionic
+
+  - server_name: "kafka5-production"
+    server_instance_type: t3.medium
+    network_tier: "db-private"
+    az: "c"
+    volume_size: 300
+    volume_encrypted: no
```

Comment:
I should have caught this earlier, but it would have been nice to switch this to `volume_encrypted: yes` which would use native aws encryption instead of ecryptfs. Next time we rebuild these we should do that.